File: CodeForces/problems/469A.py
n = int(input())
levs = set(range(1, n + 1))

vals1 = set(map(int, input().split()))
vals2 = set(map(int, input().split()))

vals1.discard(0)
vals2.discard(0)

if vals1 | vals2 == levs:
    print("I become the guy.")
else:
    print("Oh, my keyboard!")

File: CodeForces/problems/469A.cpp
#include <bits/stdc++.h>
using namespace std;

#define fastio                                                                 \
  ios::sync_with_stdio(0);                                                     \
  cin.tie(0);

int main() {
  fastio;
  int ceiling;
  cin >> ceiling;
  vector<bool> levels(ceiling, false);

  int p, s;
  cin >> p;
  for (int i = 0; i < p; i++) {
    int x;
    cin >> x;
    levels[x - 1] = true;
  }
  cin >> s;
  for (int i = 0; i < s; i++) {
    int x;
    cin >> x;
    levels[x - 1] = true;
  }

  cout << (count(levels.begin(), levels.end(), false) ? "Oh, my keyboard!"
                                                      : "I become the guy.")
       << endl;
}
